packages/kit/src/runtime/server/endpoint.js

```diff
@@ -45,22 +45,29 @@ export function is_text(content_type) {
 export async function render_endpoint(event, mod, options) {
 	const method = normalize_request_method(event);
```

Comment:
i'm 95% certain we can rely on `event.request.method` being uppercase already, since that's what `Request` mandates

```suggestion
	const { method } = event.request;
```

Reply:
(suggestions I can't make because they're outside the diff and GitHub is a nuisance: removing the `normalize_request_method` import at the top of the file, and removing the `event.request.` on line 78)